src/main/java/ru/practicum/shareit/request/model/ItemRequest.java

```diff
@@ -6,7 +6,7 @@
 import ru.practicum.shareit.user.model.User;
 
 import javax.persistence.*;
-import java.time.LocalDate;
+import java.time.LocalDateTime;
 
 @Entity
```

Comment:
Удалить @ToString.
@ToString includes lazy loaded fields and/or associations. This can cause performance and memory consumption issues. 
Аналогично для остальных энтитей